### DISCUSSION TITLE
Major Progress on FPS Game

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#119] [Show and tell]
Menus, Health Bars, Better Enemy AI...

https://user-images.githubusercontent.com/43304488/108939175-4e05d180-761f-11eb-987a-69b72de5fbdd.mp4

(Video is a bit deep fried/low quality due to compression.)

Link to game: https://n8python.github.io/dualDuel/

**Comment by 201flaviosilva** [comment]
Very nice, I liked to play, but I think the sensitivity of the mouse is too sensitive :)

**Reply by N8python** [reply]
Thank you! I will be adding a sensitivity setting soon.
